app/src/main/res/values/strings.xml

```diff
@@ -719,12 +719,12 @@
     <string name="allow_routine_ops">Allow routine ops</string>
     <string name="profile_state">Profile state</string>
     <string name="allow_routine_ops_msg">Allow the profile to be used in the routine operations</string>
-    <string name="profile_disable_msg">If enabled, apps will be disabled/enabled based on the value of the state</string>
-    <string name="profile_force_stop_msg">If enabled, apps will be force-stopped</string>
-    <string name="profile_clear_cache_msg">If enabled, cache will be cleared from the apps</string>
-    <string name="profile_clear_data_msg">If enabled, data will be cleared from the apps</string>
-    <string name="profile_block_trackers_msg">If enabled, trackers will be blocked from apps</string>
-    <string name="profile_backup_apk_msg">If enabled, apk files will be backed up</string>
+    <string name="profile_disable_msg">Disables or enables apps based on state value</string>
```

Comment:
I don't think I've used third-person anywhere else.

Reply:
It is that way all the way down here.
Is "value" needed?
"their state" might be easier.

Reply:
I agree.